docs/content/users/quickstart.md

```diff
@@ -177,48 +179,45 @@ Here's a quickstart instructions for a number of different environments:
         ddev drush uli
         ddev launch
         ```
-        
-        Note that as Drupal 10 moves from alpha to beta and then release, you'll want to change the tag from `^10@alpha` to `^10@beta` and then `^10`.
+
+        !!!tip
+            As Drupal 10 moves from alpha to beta and then release, you’ll want to change the tag from `^10@alpha` to `^10@beta` and then `^10` respectively.
```

Comment:
Please change the line above to 
`        ddev composer create --no-install drupal/recommended-project:^10@beta`

```suggestion
            As Drupal 10 moves from beta and to release, you’ll want to change the tag from `^10@beta` to `^10`.
```